Automattic/jetpack
# My Jetpack: Add products abstraction

Adds the abstraction to handle other stand-alone plugins products and also Hybrid products like Backup

#### Changes proposed in this Pull Request:

* Creates abstraction for simple and hybrid products

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Run `jetpack install -all` 
* Run `jetpack build pacakges/my-jetpack`
* As CRM and Backup are not implemented in the UI yet, the only thing you can test for now is if Boos is still working as expected
* Activate and Deactivate Boost from the My Jetpack page and confirm it works
